src/script/main/app.ts

```diff
@@ -543,19 +543,27 @@ class App {
         }
 
         default: {
-          this.logger.error(`Caused by: ${(error ? error.message : undefined) || error}`, error);
+          this.logger.warn(`Caused by: ${error?.message || 'unknown'}`, error);
+
+          const isNetworkError = error.name === 'NetworkError';
+          if (isNetworkError) {
+            this.logger.warn('Backend is not reachable. Reloading page.', error);
+            window.location.reload();
```

Comment:
As already discussed, this might DDoS our backend in case of an outage and the backend is coming up again.